src/env_loader.zig

```diff
@@ -1270,7 +1270,7 @@ pub const Map = struct {
     }
 
     pub fn remove(this: *Map, key: string) void {
-        this.map.remove(key);
+        _ = this.map.swapRemove(key);
```

Comment:
This was a doozy, seems that nowhere `map.remove` was being called so this error never surfaced. Not sure why the static analyzer does not pick it up, but Zig's ArrayHashMap does not have a `remove` method, only a few others and I chose [swapRemove](https://ziglang.org/documentation/master/std/#std.array_hash_map.ArrayHashMap.swapRemove) - do you agree @dylan-conway

Reply:
static analysis is lazy, it does not visit unused things. swapRemove should be fine here unless the ordering of the env keys matters (i dont think the order matters, so this is ok)

Reply:
Thank you @paperdave